### PR TITLE
30 user env

### PR DESCRIPTION
**Pull request by sukeshni**

### CODE REVIEW

**Comment by sukeshni** [comment]
@fanwashere 
For res1 body of response is empty.
For Constants.scala,
Added environment variables 
- user 
- userRepo  
  Please review.

**Comment by sukeshni** [comment]
For it to work you have to add
export USER=[Your User Name]
export USER_REPO=[Your Repo name that exists]

**Comment by takayukioda** [comment]
@sukeshni
Using environment variable called USER isn't a good idea, since it is used by a system. 
I prefer to set prefix for variables. In this case,  GITHUB_USER and GITHUB_REPO would be fine.

**Comment by sukeshni** [comment]
@shunjikonishi Yes. Sounds good. Will rename that.

**Comment by sukeshni** [comment]
@da0shi Changed :)
Please review

**Comment by takayukioda** [comment]
for environmentvariables, LGTM
@fanwashere please review the rest.

File: src/test/scala/Constants.scala [hunk 1]
```diff
@@ -27,13 +25,18 @@ trait Constants {
       println(v)
     }
   }
-  protected val otherUser = "shunjikonishi" 
+
+  protected val user = sys.env("GITHUB_USER")
+  protected val userRepo = sys.env("GITHUB_REPO")
```
Comment (fanwashere) [inline src/test/scala/Constants.scala]: Not sure if userRepo is necessary. The repo we used to test the API won't change often if at all. It is already defined in line 17.

**Comment by fanwashere** [comment]
LGTM

**Comment by shunjikonishi** [comment]
@sukeshni @fanwashere 

I think README.md in deep directory might be difficult to find.
ROOT/test.md might be better.

Otherwise LGTM

**Pull request merged**